modules/serverless-upgrade-sub-channel.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// serverless/installing_serverless/upgrading-serverless.adoc
+
+[id="serverless-upgrade-sub-channel_{context}"]
+== Upgrading the Subscription Channel
+
+To upgrade to the latest version, you must update the *Subscription Update Channel* to `4.4`.
+
+[NOTE]
+====
+If you have selected Manual updates, you will need to complete additional steps after updating the channel as described in this guide. The Subscription’s upgrade status will remain *Upgrading* until you review and approve its Install Plan. Information about the Install Plan can be found in the {product-title} Operators documentation.
+====
+
+.Prerequisites
+* You have installed a previous version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
+* You have logged in to the {product-title} web console.
+
+.Procedure
+
+. Select the `openshift-operators` namespace in the {product-title} web console.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Select the *{ServerlessOperatorName} Operator*.
+. Click *Subscription* → *Channel*.
+. In the *Change Subscription Update Channel* window, select `4.4`, and then click *Save*.
+. Wait until all pods have been upgraded in the `knative-serving` namespace and the KnativeServing custom resource reports the latest Knative Serving version.
+
+.Verification steps
+
+To verify that the upgrade has been successful, you can check the status of pods in the `knative-serving` namespace, and the version of the KnativeServing CR.
```

Comment:
Maybe s/KnativeServing/`KnativeServing`?

Reply:
Comment from Alex:
if we go by https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting then it would be without backticks, unless referring to a specific field/variable.

I also got this feedback, i.e. remove the backticks, on another review recently, so I'll leave them off if that's OK :)